packages/client/src/styles/globals.css

```
@@ -2613,6 +2613,24 @@ strong {
 }
+
+@media (max-width: 767px) {
+  .mari-message-content .mari-md-inline-code {
+    word-break: break-all;
+  }
+
+  .mari-message-content .mari-md-codeblock {
+    overflow-x: hidden;
+    overflow-wrap: anywhere;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)